common/source/docs/common-wiki-editing-setup.rst

```diff
@@ -30,6 +30,17 @@ First, the wiki should be forked and cloned much like :ref:`developers do for th
    .. code-block:: bash
 
        git clone https://github.com/YOURID/ardupilot_wiki.git
+
+Next, update your fork of the repository, both locally and on GitHub
```

Comment:
```suggestion
Be sure to keep your fork of the repository updated, both locally, and on GitHub both as you install and as you develop new contributions. Be sure to create a working branch locally when you start to make changes, pushing that branch up to your GitHub repo, and then making a PR (pull-request from there to the Wiki repo), in a manner similar to making :ref:`code PRs <submitting-patches-back-to-master>`
```

Reply:
Hmm...I thought it was weird as well. I was working on a fresh VMWare VM with Ubuntu 20.04. After it happened the first time, I removed the repository from github, removed it from the VM, re-forked it from github, and then noticed this:

https://cdn.discordapp.com/attachments/718666106625327175/802715354014416986/Capture.PNG

I looked around on github and I couldn't find anything related to forking different versions? Any further ideas for troubleshooting?

Reply:
I dont think you can fork anything but the current state of the repo using the FORK button in GITHUB...possibly you were not using the master branch of that forked repo but rather on one of the other branches (which should not be there anyway....I need to go clean them up again since there are several that people added since I last cleaned up)....

Reply:
Hmm...okay, that must have been the case!